### PR TITLE
Add support to build aarch64 wheels

### PR DESCRIPTION
**Pull request by tbbharaj**
In reference to issue https://github.com/python/typed_ast/issues/180, created this PR to add support for aarch64 wheels.
Happy to address any feedbacks!
Successful builds: https://github.com/tbbharaj/typed_ast/actions/runs/1546050914

Thank you

### CODE REVIEW

**Comment by the-knights-who-say-ni** [comment]
Hello, and thanks for your contribution!

I'm a bot set up to make sure that the project can legally accept this contribution by verifying everyone involved has signed the [PSF contributor agreement](https://www.python.org/psf/contrib/contrib-form/) (CLA).

## Recognized GitHub username

We couldn't find a [bugs.python.org](https://bugs.python.org/) (b.p.o) account corresponding to the following GitHub usernames:

@tbbharaj

This might be simply due to a missing "GitHub Name" entry in one's b.p.o account settings. This is necessary for legal reasons before we can look at this contribution. Please follow [the steps outlined in the CPython devguide](https://devguide.python.org/pullrequest/#licensing) to rectify this issue.




You can [check yourself](https://check-python-cla.herokuapp.com/) to see if the CLA has been received.

Thanks again for the contribution, we look forward to reviewing it!

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -26,7 +25,7 @@ jobs:
       - name: Build wheels
         env:
           CIBW_BUILD: "cp${{ matrix.python-version }}-*"
-          CIBW_SKIP: "*-manylinux_i686 *-win32"
+          CIBW_SKIP: "*-manylinux_i686 *-win32 *-macosx_arm64"
```
Comment (hauntsaninja) [inline .github/workflows/build.yml:28]: Is this intentional? We still want arm64 wheels for macOS right?

**Review by hauntsaninja — COMMENTED** [review]
Thank you! Sorry, between waiting for the CLA business and the holidays this got a little bit lost. Just had one question :-)

**Comment by scottshambaugh** [comment]
I think this is related, I'm seeing this issue pop up on CI for python 3.10.1 on Windows 10 and Mac OSX. `typed-ast 1.4.3` isn't found on PyPi, and building fails for those two configurations. Python 3.7-3.9 work fine, and Ubuntu works fine.

Here's a representative run which was passing about a month ago, but failing now: https://github.com/scottshambaugh/monaco/runs/4847920477?check_suite_focus=true

File: .github/workflows/build.yml (hunk 1 above)
Comment (tbbharaj) [reply .github/workflows/build.yml]: Yes, added it back to be keep it same as before

**Comment by hauntsaninja** [comment]
Hmm, looks like aarch64 build hung on Python 3.10. Let me try kicking it, in case it was a fluke.

**Comment by tbbharaj** [comment]
> Hmm, looks like aarch64 build hung on Python 3.10. Let me try kicking it, in case it was a fluke.

@hauntsaninja On my branch I was able to get aarch64 build on python 3.10 successfully. https://github.com/tbbharaj/typed_ast/actions/runs/1713796535

**Comment by tbbharaj** [comment]
@hauntsaninja Does this look good to merge?

**Comment by hauntsaninja** [comment]
Yup, thanks for your patience. It's a little unfortunate that the build seems flakey, but we'll see how bad of a problem that is going forward. I'll release a 1.5.2 with aarch64 wheels.

**Comment by hauntsaninja** [comment]
Thanks, 1.5.2 is out!

**Pull request merged**